Sources/BraveStrings/BraveStrings.swift

```diff
@@ -60,8 +60,8 @@ extension Strings {
   public static let sendReportButtonTitle = NSLocalizedString("SendReportButtonTitle", tableName: "BraveShared", bundle: .module, value: "Send Report", comment: "Used as a button label for crash dialog prompt")
   public static let alwaysSendButtonTitle = NSLocalizedString("AlwaysSendButtonTitle", tableName: "BraveShared", bundle: .module, value: "Always Send", comment: "Used as a button label for crash dialog prompt")
   public static let dontSendButtonTitle = NSLocalizedString("DontSendButtonTitle", tableName: "BraveShared", bundle: .module, value: "Don’t Send", comment: "Used as a button label for crash dialog prompt")
-  public static let restoreTabOnCrashAlertTitle = NSLocalizedString("RestoreTabOnCrashAlertTitle", tableName: "BraveShared", bundle: .module, value: "Well, this is embarrassing.", comment: "Restore Tabs Prompt Title")
-  public static let restoreTabOnCrashAlertMessage = NSLocalizedString("RestoreTabOnCrashAlertMessage", tableName: "BraveShared", bundle: .module, value: "Looks like Brave crashed previously. Would you like to restore your tabs?", comment: "Restore Tabs Prompt Description")
+  public static let restoreTabOnCrashAlertTitle = NSLocalizedString("RestoreTabOnCrashAlertTitle", tableName: "BraveShared", bundle: .module, value: "Brave Closed Unexpectedly", comment: "Restore Tabs Prompt Title")
+  public static let restoreTabOnCrashAlertMessage = NSLocalizedString("RestoreTabOnCrashAlertMessage", tableName: "BraveShared", bundle: .module, value: "Unfortunately, Brave crashed or did not close properly. \nRestore your tabs?", comment: "Restore Tabs Prompt Description")
```

Comment:
You do not need the empty space, the other next sentence is already starting in new line.